src/bptl/work_units/xential/tasks.py

```diff
@@ -1,28 +1,38 @@
+import json
+import uuid
+from xml.dom import minidom
+
+from django.conf import settings
+from django.contrib.sites.models import Site
+
+from rest_framework.reverse import reverse
+
 from bptl.tasks.base import BaseTask, check_variable
 from bptl.tasks.registry import register
 
-from .client import get_client, require_xential_service
+from .client import XENTIAL_ALIAS, get_client, require_xential_service
+from .models import XentialTicket
 
 
 @register
 @require_xential_service
 def start_xential_template(task: BaseTask) -> dict:
     """
     Run Xential template with requested variables.
-    The returned value is either buildId of created document (in case it is a silent template)
-    or the url to the page where the user can manually fill in the input fields (in case it is
-    an interactive template).
+    If the ``interactive`` task variable is:
+    * ``"True"``: it returns a URL in ``bptlDocumentUrl`` for building a document interactively
+    * ``"False"``: it returns an empty string in ``bptlDocumentUrl``
```

Comment:
We should be able to use the native types rather than using strings for True/False: https://docs.camunda.org/manual/latest/user-guide/process-engine/variables/#supported-variable-values

Reply:
I also changed this to be able to use a boolean:
https://github.com/GemeenteUtrecht/bptl/blob/master/src/bptl/tasks/base.py#L13
because otherwise if `variable[name]` is `False`, then we get an error.